sources/packages/web/src/views/institution/designations/ManageDesignations.vue

```diff
@@ -8,10 +8,18 @@
       />
     </template>
     <body-header
-      title="Designation agreements"
-      subTitle="Ensure you have an active designation to administer student financial aid."
+      title="Designation requests"
       :recordsCount="designations.length"
     >
+      <template #subtitle>
+        Ensure you have an approved designation to administer financial aid to
+        students
+        <tooltip-icon
+          >To request a designation, you must have the user role of a Legal
+          Signing Authority to access the request button here.</tooltip-icon
+        >
+      </template>
+
```

Comment:
Just a thought but instead of removing the button (`v-if="isLegalSigningAuthority"`), we could just disable it (`":disabled="!isLegalSigningAuthority"`) to allow the user without the role (could be by mistake) to have a better idea about what the tooltip is referring to.